docs/sources/reference/playlist.md

```diff
@@ -11,59 +11,95 @@ weight = 4
 
 # Playlist
 
-The Playlist is a special type of Dashboard that rotates through a list of Dashboards. A Playlist can be a great way to build situational awareness, or just show off your metrics to your team or visitors.
+The Playlist is a special type of Dashboard that rotates through a list of Dashboards. A Playlist can be a great way to build situational awareness, and show off your metrics to your team or visitors.
 
-Since Grafana automatically scales Dashboards to any resolution they're perfect for big screens!
+Grafana automatically scales Dashboards to any resolution, which makes them perfect for big screens.
+
+Access the Playlist feature from Grafana's side menu, in the Dashboard submenu.
 
 ## Create a playlist
 
 {{< docs-imagebox img="/img/docs/v50/playlist.png" max-width="25rem" class="docs-image--right">}}
 
-The Playlist feature can be accessed from Grafana's sidemenu, in the Dashboard submenu.
+To create a playlist: 
+
+1. Click the __New playlist__ button.
+2. Name your playlist in the __Name__ field.
+3. Enter a time interval in the __Interval__ field.
+
+The time interval is the amount of time for Grafana to stay on a particular Dashboard before advancing to the next one on the Playlist.
```

Comment:
Looks like this refers to a specific playlist, not the actual feature, so probably shouldn't be capitalized here?